packages/dictionary/src/utils/typeUtils.ts

```diff
@@ -26,6 +26,44 @@
  */
 export const asArray = <T>(val: T | T[]): T[] => (Array.isArray(val) ? val : [val]);
 
+/**
+ * Given a predicate function that checks for type `T`, this will create a new predicate funcion that
+ * will check if a value is of type `T[]`.
+ *
+ * @example
+ * // Create type and predicate for `Person`:
+ * type Person = { name: string; age: number };
+ * const isPerson = (value: unknown): value is Person =>
+ * 	!!value &&
```

Comment:
nitpicking on the example, for the sake of documentation simplicity (this line adds superfluous cognitive burden for a mere example)... type of falsey values will never be 'object', the first check, and therefor `!Person`.

```suggestion
```

Reply:
`null` is falsey and is type `object`

Reply:
fds! you're right!
in that case precise intent expression is always preferable for its declarativeness
```suggestion
 * 	value != null &&
```

Reply:
Yeah thats probably clearer. I'll start using that instead.